includes/classes/class-ct-db-registration.php

```diff
@@ -288,14 +288,20 @@ public function registration_form_fields() {
 							<input type="submit" value="<?php _e( 'Register Your Account', 'wp-discussion-board' ); ?>"/>
 						</p>
 
+						<p>
+							<span id="ctdb_user_form-response" class="validation-response"><small>Please make sure all required fields are filled out.</small></span>
```

Comment:
Can you add translation to this string with escaping. So using the following:

`<?php esc_html_e( 'Please make sure all required fields are filled out.', 'wp-discussion-board' ); ?>`